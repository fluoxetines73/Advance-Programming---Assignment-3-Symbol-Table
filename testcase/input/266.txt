BEGIN
INSERT b number
RPRINT
INSERT a string
PRINT
END