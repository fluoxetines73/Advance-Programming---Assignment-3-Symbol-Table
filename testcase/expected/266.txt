success
b//1
success
b//1 a//1